packages/dashboard/.env.example

```diff
@@ -1,19 +1,20 @@
 # Client Side Variables -- Anything using NEXT_PUBLIC_ will be readable in the client
```

Comment:
I think we should also have an example to ganache maybe .env.ganache.example ? Then this file doesn't need all the ethereum configs in it